v03_pipeline/lib/tasks/base/base_project_info_params.py

```diff
@@ -0,0 +1,11 @@
+import luigi
+import luigi.util
+
+from v03_pipeline.lib.tasks.base.base_loading_run_params import BaseLoadingRunParams
+
+
+@luigi.util.inherits(BaseLoadingRunParams)
+class BaseLoadingRunWithProjectInfoParams(luigi.Task):
```

Comment:
@bpblanken how do you feel about this naming...

Reply:
I think it's ok!  One sneaky thing, we've done work to make the pedigree paths a computable function of `project_guid` so this isn't going to be as necessary in the future.  The right thing for now though!